### PR TITLE
CSS, etc.

### PR DESCRIPTION
**Pull request by mnijmeh11**

### CODE REVIEW

**Comment by B3nD3lan3y** [comment]
Hey @mnijmeh11, did you mean to leave this open?

**Comment by B3nD3lan3y** [comment]
it's a merge into your own branch (not into dev or master), so you're welcome to merge it yourself, without approval.

**Comment by chaneberg** [comment]
This is what these changes look like on my browser:

capture

This is the play list page, not the play detail page.

File: statler_site/templates/mobile/css/style.css [hunk 1]
```diff
@@ -0,0 +1,841 @@
...
+.social.gplus{
+	background-position: -440px 0px;
+}
+.social.gplus:hover{
+	background-position: -440px -40px;
+}
+.social.pintrest{
+	background-position: -480px 0px;
+}
+.social.pintrest:hover{
```
Comment (chaneberg) [inline statler_site/templates/mobile/css/style.css:446]: We aren't using any of this CSS and it should be removed.

File: statler_site/templates/play-list-page.html [hunk 2]
```diff
@@ -1,61 +1,352 @@
...
+          );
+          $('#details').append(
+            $("<p/>").append("Directed by: ", result.director),
+            $("<p/>").append("Actors: ", result.actors),
+            $("<p/>").append("Showtimes: ", result.show_times)
+          );
+          // fill in reviews
+          $.each(result.reviews, function(index, review) {
+            $("#review-list").append(
+              $("<li/>").append(review.text)
```
Comment (chaneberg) [inline statler_site/templates/play-list-page.html:320]: It looks like all the code in the play list page has been replaced with the code from the play details page.

File: statler_site/templates/play-list-page.html.bak [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+              <script  src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+              <!-- Latest compiled and minified CSS -->
+              <meta  name="viewport"  content="width=device-width, initial-scale=1, maximum-scale=1">
+              <script  src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+              <style>
+.background
+{
+knob
+background: rgba(76,76,76,1);
+background: -moz-linear-gradient(left, rgba(76,76,76,1) 0%, rgba(89,89,89,0.99) 12%, rgba(102,102,102,0.97) 25%, rgba(71,71,71,0.95) 39%, rgba(44,44,44,0.94) 50%, rgba(0,0,0,0.94) 51%, rgba(17,17,17,0.93) 60%, rgba(43,43,43,0.91) 76%, rgba(28,28,28,0.89) 91%, rgba(19,19,19,0.88) 100%);
```
Comment (chaneberg) [inline statler_site/templates/play-list-page.html.bak:17]: CSS shouldn't be inline.

Comment (bencdelaney) [inline statler_site/templates/play-list-page.html.bak:17]: Also, I don't think this .bak file should be here.

**Comment by bencdelaney** [comment]
There are a number of issues here, and I think we ought to talk as a group (#112) prior to merging new CSS. Let's close this pull request for now, and revisit the issue on Thursday at the latest.

**Pull request closed without merging**